python/ray/tests/BUILD

```diff
@@ -25,7 +26,7 @@ py_test_module_list(
     "test_error_ray_not_initialized.py",
     "test_gcs_fault_tolerance.py",
     "test_iter.py",
-    "test_resource_demand_scheduler.py",
```

Comment:
Did a tiny bit of reorganizing here since I think "R" comes after "J" in the alphabet :p